tests/end2end-tests/custom-gnn/run_test.sh

```diff
@@ -24,21 +23,21 @@ error_and_exit () {
 	fi
 }
 
-python3  ${LAUNCH_PATH} \
+python3 -m graphstorm.run.launch \
         --workspace /data \
         --num_trainers ${NUM_TRAINERS} \
         --num_servers ${NUM_SERVERS} \
         --num_samplers ${NUM_SAMPLERS} \
         --part_config ${PART_CONFIG} \
         --ip_config ${IP_CONFIG} \
         --ssh_port 2222 \
-        "python3 ${GSF_HOME}/tests/end2end-tests/custom-gnn/train.py \
+        ${GSF_HOME}/tests/end2end-tests/custom-gnn/train.py \
         --ip-config ${IP_CONFIG} \
         --part-config ${PART_CONFIG} \
```

Comment:
Should not specify part_config and ip_config twice. And better to be consistent with the "_" and "-" in arguments.
       --part_config ${PART_CONFIG} \
        --ip_config ${IP_CONFIG} \
        --ip-config ${IP_CONFIG} \
        --part-config ${PART_CONFIG} \